ingeniamotion/fsoe.py

```diff
@@ -353,19 +354,8 @@ class FSoEMaster:
     SAFETY_ADDRESS_REGISTER = EthercatRegister(
         idx=0x4193, subidx=0x00, dtype=RegDtype.U16, access=RegAccess.RW
     )
-    SAFE_INPUTS_MAP_REGISTER = EthercatRegister(
-        identifier="SAFE_INPUTS_MAP",
-        idx=0x46D2,
-        subidx=0x00,
-        dtype=RegDtype.U16,
-        access=RegAccess.RW,
-    )
-    SS1_TIME_TO_STO_REGISTER = EthercatRegister(
-        identifier="SS1_TIME_TO_STO",
-        idx=0x6651,
-        subidx=0x01,
-        dtype=RegDtype.U16,
-        access=RegAccess.RW,
+    CONFIGURED_MODULE_IDENT_1_REGISTER = EthercatRegister(
```

Comment:
I think this register should read from dictionary with its UID. It should be in dictionary V3, and in dictionary V2 should be added as monitoring register, PDO registers, etc..

Reply:
SAFETY_ADDRESS_REGISTER  too